docs/user/openstack/known-issues.md

```diff
@@ -0,0 +1,15 @@
+# Known Issues and Workarounds
+
+During this release, we are shipping with a few known issues. We are documenting them here, along with whatever workarounds we are aware of, and have attached the links to where the engineering team is tracking the issues. As changes occur, we will update both this document, and the issue trackers with the lates information.
+
+## Long Hostnames
+
+The cluster sets the hostname of servers to `<clusterName> Workstation-<clusterName>-<clusterID>-master-<#>`. There is a character limit of 64, so creating a cluster name that is too long could result in an installer crash. In order to prevent this from happening, please restrict the length of the `metadata.clusterName` field to 18 characters as a baseline. This value is dependant on the number of masters you have, since the masters are numbered. For each multiple of 10, subtract 1 from the max number of characters. For example, if you have 25 masters, your cluster name will be restricted to 17 characters. The installer will validate this in your install config and throw an error to prevent you from triggering this install time bug. This is being tracked in this [github issue](https://github.com/openshift/installer/issues/2243).
```

Comment:
Not sure but max -> maximum looks more accurate here.

Reply:
It's not the hostname of servers, but the service name in mDNS.
The hostnames are currently in the form `<clusterID>-master-<#>`.